app/helpers/application_helper/toolbar/ansible_credential_center.rb

```diff
@@ -2,7 +2,7 @@ class ApplicationHelper::Toolbar::AnsibleCredentialCenter < ApplicationHelper::T
   button_group('ansible_repository', [
     select(
       :ansible_credential_configuration,
-      'fa fa-cog fa-lg',
+      '',
```

Comment:
I'd rather go with `nil` instead of allocating an empty string each time.

Reply:
@skateman Updated